vscode/vscode.md

```diff
@@ -0,0 +1,103 @@
+This section outlines how to set up the application in the VS CODE IDE.
+
+## Prerequisites
+
+* Java SE 8, Java SE 11 or Java SE 17 is required.
+* Payara Server 5 is required. You can download Payara Server 5 from [here](https://www.payara.fish/downloads/).
+* The VS CODE IDE is required. You can download the VS CODE from [here](https://code.visualstudio.com/download).
+
+## Download
+
+[Download](https://github.com/eclipse-ee4j/cargotracker/archive/master.zip) the source code zip file and expand it somewhere in your file system. Note that this is a Maven project.
+
+## VSCODE IDE Setup
+
+Before going through this tutorial, you must install the following extensions from the Visual Studio Code marketplace:
+
+* [Java Extension Pack, by Microsoft](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack)

```

Comment:
The Java extension should be considered a pre-requisite. It's OK to keep the instructions to add Payara Tools.

Reply:
In pre-requisite I mentioned it. should I delete this line?